src/codegen/type/varchar_type.cpp

```diff
@@ -303,6 +303,32 @@ struct RTrim : public TypeSystem::BinaryOperatorHandleNull {
   }
 };
 
+struct Substr : public TypeSystem::NaryOperator {
```

Comment:
Can you add some comments somewhere that outlines what the expected arguments to the function are?

Reply:
Sure, just added some comments

Reply:
Just added some comments